app/services/gtfs_graph.rb

```diff
@@ -127,54 +134,56 @@ def log(msg)
 
   def load_tl_stops
     # Merge child stations into parents.
-    log "  merge stations"
-    stations = Hash.new { |h,k| h[k] = [] }
+    log '  merge stations'
+    stations = Hash.new { |h, k| h[k] = [] }
     @gtfs.stops.each do |stop|
       stations[@gtfs.stop(stop.parent_station) || stop] << stop
     end
     # Merge station/platforms with Stops.
-    log "  stops"
-    stations.each do |station,platforms|
+    log '  stops'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.